packages/core/integration-tests/test/pwa-manifest.js

```diff
@@ -0,0 +1,67 @@
+const assert = require('assert');
+const path = require('path');
+const {
+  bundle,
+  assertBundles,
+  distDir,
+  outputFS
+} = require('@parcel/test-utils');
+
+describe('pwa-manifest', function() {
+  it('should support .webmanifest', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/pwa-manifest/index.html')
+    );
+
+    await assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html']
+      },
+      {
+        type: 'webmanifest',
+        assets: ['manifest.webmanifest']
+      },
+      {
+        type: 'png',
+        assets: ['icon.png']
+      },
+      {
+        type: 'png',
+        assets: ['screenshot.png']
+      },
+      {
+        type: 'js',
+        assets: ['serviceworker.js']
+      }
+    ]);
+  });
+
+  it('should support .json webmanifest', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/pwa-manifest-json/index.html')
+    );
+
+    await assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html']
+      },
+      {
+        type: 'webmanifest',
+        assets: ['manifest.json']
```

Comment:
in `BundleGraph` we have `manifest.json`, but then in `dist` - `manifest. webmanifest` - is that a problem?